ATS Routes/RNAV/Upper/UY53.txt

```diff
@@ -1,4 +1,6 @@
+DTY   DTY   SAPCO SAPCO
+SAPCO SAPCO NUGRA NUGRA
```

Comment:
```suggestion
;Route Break
```
![image](https://github.com/user-attachments/assets/d14aaec3-e93d-4b4e-ae23-1db2083278b4)
![image](https://github.com/user-attachments/assets/a10a097c-c332-4bf9-a042-23f8b1879d80)
SAPCO-NUGRA is not on UY53 per the above.